### ISSUE TITLE
low performance: Check AUgemntation and Normalization

### ISSUE THREAD

**Issue by initze** [initze/thaw-slump-segmentation#78]
### Issue
* Model performance rather low on validation dataset:

### Solutions
* check for better data normalization
  * train and val
* double check Augmentation functionality
  * torchvision vs albumentations

**Comment by initze** [comment]
Normalization is now implemented

**Comment by initze** [comment]
went back to albumentations: https://github.com/initze/thaw-slump-segmentation/commit/56b11f8fed2b0d0afbcbdf9632e89a8611826791

* required some reshaping of target/mask/label data
* Should work fine now

**Comment by initze** [comment]
https://pytorch.org/vision/stable/transforms.html#transform-classes-functionals-and-kernels
